products/iap/terraform.yaml

```diff
@@ -129,6 +129,27 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           backend_service_name: "backend-service"
           http_health_check_name: "health-check"
         primary_resource_name: "fmt.Sprintf(\"backend-service%s\", context[\"random_suffix\"])"
+  TunnelInstance: !ruby/object:Overrides::Terraform::ResourceOverride
+    iam_policy: !ruby/object:Api::Resource::IamPolicy
+      exclude: false
+      method_name_separator: ':'
+      parent_resource_type: 'google_compute_instance'
+      parent_resource_attribute: 'instance'
+      fetch_iam_policy_verb: :POST
+      allowed_iam_role: 'roles/iap.tunnelResourceAccessor'
+      iam_conditions_request_type: :REQUEST_BODY
+    id_format: "projects/{{project}}/iap_tunnel/zones/{{zone}}/instances/{{name}}"
+    import_format: [
+      "projects/{{project}}/iap_tunnel/zones/{{zone}}/instances/{{name}}",
```

Comment:
Currently the handwritten has `"projects/{{project}}/iap_tunnel/zones/{{zone}}/instances/{{instance}}"` for id and `"projects/{{project}}/zones/{{zone}}/instances/{{instance}}"` for import format, so we're keeping ID format and supporting both import formats